Directory.Build.props

```diff
@@ -35,10 +35,10 @@
          * Major/Minor match Android stable API level, such as 30.0 for API 30.
          * Bump first digit of the patch version for feature releases (and reset the first two digits to 0)
     -->
-    <AndroidPackVersion>35.0.0</AndroidPackVersion>
-    <AndroidPackVersionSuffix>rc.2</AndroidPackVersionSuffix>
+    <AndroidPackVersion>35.0.1</AndroidPackVersion>
+    <AndroidPackVersionSuffix>rtm</AndroidPackVersionSuffix>
     <IsStableBuild>false</IsStableBuild>
-    <IsStableBuild Condition=" '$(AndroidPackVersionSuffix)' == 'rtm' ">true</IsStableBuild>
+    <IsStableBuild Condition=" '$(AndroidPackVersionSuffix)' == 'rtm' and $(XAVersionBranch.StartsWith('release/')) ">true</IsStableBuild>
```

Comment:
I think we'll want to move this into the `GetXAVersionInfo` target now, since it is now tied to `$(XAVersionBranch)`